### PR TITLE
Security: Remove all plain text password logic and ui

### PR DESCRIPTION
**Pull request by JohnMcLear**
Fixes https://github.com/ether/etherpad-lite/issues/230

This doesn't deprecate.
It doesn't gracefully remove the API methods from being available
It doesn't publish an API version.

It is a pure cut off.  If you use these API endpoints, it will screw you over.

### CODE REVIEW

**Comment by rhansen** [comment]
Please rebase onto current develop.

**Comment by rhansen** [comment]
> Please rebase onto current develop.

I rebased your branch onto develop and force-pushed to your fork.

File: src/templates/pad.html [hunk 1]
```diff
@@ -91,18 +91,6 @@
...
-          <form class='passForm' method='POST'>
-            <input id='passwordinput' type='password' name='password'><input type='submit' value='Submit'>
-          </form>
-        </div>
-        <div id="permissionDenied">
-          <p data-l10n-id="pad.permissionDenied">You do not have permission to access this pad</p>
-        </div>
-        <div id="wrongPassword">
-          <p data-l10n-id="pad.wrongPassword">Your password was wrong</p>
-        </div>
```
[the remark this answers is not in the document]
Comment (JohnMcLear) [reply src/templates/pad.html:105]: No.  I don't think we ever exposed them either, note the lack of ejs template content..

[the remark this answers is not in the document]
Comment (JohnMcLear) [reply src/templates/pad.html:105]: agreed.

[the remark this answers is not in the document]
Comment (JohnMcLear) [reply src/templates/pad.html:105]: Fixed in https://github.com/ether/etherpad-lite/pull/4178/commits/0bb8c474eb33ccd2b2b8d4e0ccf5fed6eb320ca3 -- updating en.json should hopefully be enough as TW can handle rest internally.

**Comment by JohnMcLear** [comment]
Ready for further review.

**Comment by rhansen** [comment]
I rebased your branch and pushed some additional changes. Please take a look. If everything looks good to you, I think it's ready to squash and merge.

**Pull request merged**